hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxData.java

```diff
@@ -353,6 +353,8 @@ private static EthTxData populateLegacyEthTxData(RLPItem rlpItem, byte[] rawTx)
         byte recId = vBI.testBit(0) ? (byte) 0 : 1;
```

Comment:
I know this is a `private` method - but that just means we have no unit test for it, apparently.  Is that a good idea?  Aren't there at least 3 things in here that we could be more confident in if there was a unit test?